### DISCUSSION TITLE
show content type filtered by category

### DISCUSSION THREAD

**Discussion by mrccarvalho** [PiranhaCMS/piranha.core#1951] [Q&A]
Hi! 
I have a page with a list of a content type named StandardMovie.

What is the best way to filter the movies by category ?
Let say i have a select field with all movies categories and after choose a category u want to render/update the view with the movies that belongs to the category selected.

Thanks.

**Comment by tidyui** [comment]
The simplest way would most likely be build your movie page as an Archive and create the movies as Posts in that archive as Posts have built in features for categories, tags and getting pages lists filtered on these taxonomies.

Regards

**Comment by tidyui** [comment]
Transferring to discussions as this is not an issue.
